exercises/alphametics/alphametics_test.rb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'alphametics'
+
+# Test data version:
+# 4626617 Merge pull request #237 from moofkit/bracket_push_tests
+
+class AlphameticsTest < Minitest::Test
+  def test_solve_short_puzzle
+    expect = { 'I' => 1, 'B' => 9, 'L' => 0 }
+    actual = Alphametics.new.solve('I + BB == ILL')
+    assert_equal(expect, actual)
+  end
+
+  def test_solve_long_puzzle
+    skip
+    expect = { 'S' => 9, 'E' => 5, 'N' => 6, 'D' => 7, 'M' => 1, 'O' => 0, 'R' => 8, 'Y' => 2 }
```

Comment:
According to rubocop this line is too long. The hash was gathered from the `expected` in the JSON, and I have added some better formatting in `AlphameticsCase#expected_value`, but I was wondering about this issue. Suggestions?